recipes/minibusco/meta.yaml

```diff
@@ -1,38 +1,14 @@
 {% set name = "minibusco" %}
-{% set version = "0.2.1" %}
-{% set hash = "625ef6cff116ca28b589afd240141440d7e614aedacc586fc66d4af20c0ffec3" %}
+{% set version = "fix_bioconda_recipe" %}
```

Comment:
The package version needs to be larger then the old one. This is a string that is not guaranteed to be larger.